youtube_dl/extractor/polskieradio.py

```diff
@@ -0,0 +1,44 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+import json
+
+
+class PolskieRadioIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?polskieradio\.pl/[0-9]+/[0-9]+/Artykul/(?P<id>[0-9]+),.*'
+    _TESTS = [{
+        'url': 'http://www.polskieradio.pl/7/5102/Artykul/1587943,Prof-Andrzej-Nowak-o-historii-nie-da-sie-myslec-beznamietnie',
+        'md5': '2984ee6ce9046d91fc233bc1a864a09a',
+        'info_dict': {
+            'id': '1587943',
+            'ext': 'mp3',
+            'title': 'Prof. Andrzej Nowak: o historii nie da się myśleć beznamiętnie',
+            'description': '- O polskiej historii nie da się myśleć bez wchodzenia w ten emocjonujący spór, co było dobre, a co złe, co było błędem, a co było słuszną decyzją - mówi profesor Andrzej Nowak.',
+        }
+    }, {
+        'url': 'http://polskieradio.pl/8/295/Artykul/213411,Wokol-Historyji-o-Chwalebnym-Zmartwychwstaniu-Panskim',
+        'md5': 'e925764839c67820b4fbdcd7e2785c08',
+        'info_dict': {
+            'id': '213411',
+            'ext': 'mp3',
+            'title': 'Wokół "Historyji o Chwalebnym Zmartwychwstaniu Pańskim"',
+            'description': '„Wokół Historyji o Chwalebnym Zmartwychwstaniu Pańskim" - audycja Iwony Malinowskiej i Bogumiły Prządki z udziałem historyka literatury - profesora Krzysztofa Mrowcewicza, historyka teatru - profesora Andrzeja Kruczyńskiego, reżysera teatralnego, który wystawiał misterium Mikołaja z Wilkowiecka na scenie - Piotra Cieplaka i reżysera radiowego - Zdzisława Dąbrowskiego, który zrealizował ten utwór dla radia w 1977 roku.',
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        metadata_string = self._html_search_regex(r'data-media=\{([^\}]+)\}', webpage, 'metadata')
```

Comment:
Expand regex-group to include the two curly braces, to avoid the awkward 'plus' statement below. And use `self._parse_json()` instead of `json.loads()`.

It's better to parse the regex into `metadata` via `self._parse_json()` instead of creating an own variable for it, since you're using it only once.